Sources/Routing/Routing/TrieRouter.swift

```diff
@@ -73,26 +73,37 @@ public final class TrieRouter<Output> {
         var currentNode: RouterNode = root
 
         // traverse the string path supplied
-        search: for path in path {
+        search: for pathPart in path {
```

Comment:
Naming change though it could be reverted if you feel strongly. Having two things called `path` was making picking this section apart a little confusing.